srcpkgs/lsp-plugins/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'lsp-plugins'
+pkgname=lsp-plugins
+version=1.1.30
+revision=1
+build_style=gnu-makefile
+make_build_args="VERBOSE=1"
+hostmakedepends="pkg-config php"
+makedepends="libsndfile-devel libX11-devel libglvnd-devel lv2 cairo-devel ladspa-sdk jack-devel"
+short_desc="Collection of free plugins compatible with LADSPA, LV2 and LinuxVST"
+maintainer="Artur Sinila <freesoftware@logarithmus.dev>"
+license="LGPL-3.0-or-later"
+homepage="https://lsp-plug.in/"
+distfiles="https://github.com/sadko4u/lsp-plugins/archive/refs/tags/${version}.tar.gz"
+checksum=9cf43257729093c240375b3640b1514dff34b092b83b54a5ee68d7e8565c8f80
+
+export BUILD_ARCH="${XBPS_TARGET_MACHINE%%-musl}"
+
```

Comment:
why export? also this variable is not used anymore? why?

Reply:
@paper42 its a draft PR work in progress. So please hold your complaints until its ready for review.

You are not even Void team member but you act like you are superior than me

Reply:
> @paper42 its a draft PR work in progress. So please hold your complaints until its ready for review.
> 
> You are not even Void team member but you act like you are superior to me

I guess my reviews can sound a bit hostile sometimes, I am sorry for that, I don't mean it that way. Of course my reviews are not authoritative, I just want to help a bit. I will wait until you figure it out, I am not interested in helping you anymore.

Reply:
>  its a draft PR work in progress.

But it's being developed in the open, and pointing things out early helps with the final review later on.

> You are not even Void team member

That doesn't mean anything. Anyone can review per Void's packaging guidelines, and paper does that quite well.